CONTRIBUTING.md

```diff
@@ -52,6 +52,36 @@ The root package includes a helpful shortcut that you can use to build all packa
 pnpm validate
 ```
 
+### Filtering tests
+
+You can run tests for a specific package using a [PNPM `--filter` flag](https://pnpm.io/filtering):
+
+```sh
+pnpm --filter $PACKAGE_NAME test
+```
+
+For example to only run tests for the frames plugin:
+
+```sh
+pnpm --filter plugin-frames test
+```
+
+### Live reloading tests
+
+You can use the `test-watch` command to start a live reloading dev process that re-runs tests each time code changes. This also works with the `--filter` flag to only run tests for a specific package:
+
+```sh
+# Run tests for all packages and watch for changes:
+pnpm -r test-watch
+
+# Run tests for frames plugin and watch for changes:
+pnpm --filter plugin-frames test-watch
+```
```

Comment:
````suggestion
### Live reloading tests

You can use the `test-watch` command provided by the individual packages to start a live reloading dev process that re-runs tests each time code changes.

To select the package you want to run tests for, you have two options:

- Change your working directory to one of the packages and run the command directly:

  ```sh
  # Change to the frames plugin directory:
  cd packages/@expressive-code/plugin-frames

  # Run tests for the current package and watch for changes:
  pnpm test-watch
  ```

- Alternatively, stay in the repository root and use the `--filter` flag to run tests for a specific package:

  ```sh
  # Run tests for the frames plugin and watch for changes:
  pnpm --filter plugin-frames test-watch
  ```
````

Unfortunately, `pnpm -r test-watch` doesn't work for me. It only runs the `test-watch` command of the first package (usually `core`) and waits for changes. I've therefore changed the text to reflect that `test-watch` is only available on individual packages, and to show both ways of running the command.

Reply:
Ah yes, you’re right. I think I tested `pnpm -r test` and didn’t pay as close attention to the output when doing the same for the watch version.